analysis_options.yaml

```diff
@@ -1,5 +1,3 @@
-analyzer:
```

Comment:
why no strong?

Reply:
The analyzer thinks this setting is deprecated

Reply:
Oh! Wow...indeed!

I was thinking of something else.